app/core/core-ui/src/main/kotlin/com/hedvig/android/core/ui/compose/LogCompositions.kt

```diff
@@ -19,9 +18,7 @@ class Ref(var value: Int)
 @Suppress("NOTHING_TO_INLINE")
 @Composable
 inline fun LogCompositions(message: String) {
-  if (BuildConfig.DEBUG) {
```

Comment:
Moved this to a common module so that I could use it in Chat too to test something.
In there we have no access to `com.hedvig.app.BuildConfig`, but that's fine, we should just not call this from production code as we've already been doing